.github/workflows/nightly.yml

```diff
@@ -24,6 +24,13 @@ jobs:
             bazel-bin/tools/heir-translate --help
             bazel-bin/tools/heir-lsp --help
 
+      - name: Delete previous existing nightly tag and release
+        env:
+          GH_TOKEN: $${{ secrets.GITHUB_TOKEN }}
+        shell: bash
+        run: |
+            gh release view "nightly" && gh release delete "nightly" -y --cleanup-tag
```

Comment:
Why the `gh release view` here?

Reply:
I believe this is to make sure the release exists before trying to delete it.

Reply:
Yep! Won't happen often (maybe if we manually delete it) but it will ensure that we don't fail on delete